rfcs/README.md

```diff
@@ -0,0 +1,48 @@
+# RFCs
+
+Vector uses the RFC process to formalize discussion around _substantial_
+changes to Vector. The general goals are:
+
+* Properly spec and plan features to prevent re-work
+* Formalize discussion
+* Obtain consensus
+* Share reponsibility
+* Record the decision for posterity
+
+## Guidelines
+
+### Logical boundary
+
+Examples of changes that require a RFC:
+
+* An architectural change.
+* A data model change.
+* A new component that introduces new behavior.
+* Removing a feature.
+* Complicated tech-debt projects.
+* A substantial user-visible change.
+* A change that is questionably outside of the scope of Vector.
+
+Examples of changes that do not require a RFC:
+
+* Reorganizing code that otherwise does not change its functional behavior.
+* Quantitative improvements. Such as performance improvements.
+* Simple improvements to existing features.
+
+### Before creating a RFC
+
+1. Search Github for previous issues and RFCs on this topic.
+2. Open an issue representing the change for light discussion.
+3. In the isuee, obtain consensus that a RFC is necessary.
```

Comment:
Is `isuee` a typo?

Reply:
Yes, it’s my damn Apple keyboard. I’ll fix it.